DigitalOffice/Tests/FileServiceTests/Consumers/GetFileConsumerTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.FileService.Broker.Consumers;
+using LT.DigitalOffice.FileService.Database.Entities;
+using LT.DigitalOffice.FileService.Mappers.Interfaces;
+using LT.DigitalOffice.FileService.Models;
+using LT.DigitalOffice.FileService.Repositories.Interfaces;
+using MassTransit.Testing;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.FileServiceUnitTests.Consumers
+{
+    public class GetFileConsumerTests
+    {
+        private Mock<IFileRepository> repositoryMock;
+        private Mock<IMapper<DbFile, File>> mapperMock;
+
+        private InMemoryTestHarness harness;
+        private ConsumerTestHarness<GetFileConsumer> consumerTestHarness;
+
+        [SetUp]
+        public void SetUp()
+        {
+            mapperMock = new Mock<IMapper<DbFile, File>>();
+            repositoryMock = new Mock<IFileRepository>();
+            harness = new InMemoryTestHarness();
+            consumerTestHarness = harness.Consumer(() => new GetFileConsumer(repositoryMock.Object));
+        }
+
+        [Test]
+        public async Task ShouldConsumeAndPublishWhenRequestIsValidAndRepositoryReturnsSomething()
+        {
```

Comment:
ReturnsSomething?)

Reply:
NUnit supports only async test methods that return Task or Task<T>